.github/workflows/pr-verifier.yaml

```diff
@@ -10,14 +10,34 @@ on:
 
 jobs:
   verify:
-    name: verify PR contents
+    name: verify PR title
```

Comment:
I would keep the original title, this job has multiple steps of which the title is only one. Currently the only one, but still.

Reply:
fair enough!